### PR TITLE
[HeatMapSeries] Update documentation

### PR DESCRIPTION
**Pull request by seijikun**
Updated HeatMapSeries documentation with two examples:
One showing a simple usage example
And a more complex example using categories.

### CODE REVIEW

**Comment by objorke** [comment]
Great examples :-)
The example code should also be added to the documentation-examples repository.

File: models/series/HeatMapSeries.rst [hunk 1]
```diff
@@ -28,9 +25,159 @@ To show the x and y values with one digit, use the format string ``"{2:0.0},{4:0
...
+				//WeekDay-Axis (Bottom)
+				modelP1.Axes.Add(new CategoryAxis
+				{
+					Position = AxisPosition.Bottom,
+					//Key used for specifying this axis in the HeatMapSeries
+					Key = "WeekDayAxis",
+					//Property of the items in ItemSource that should be used as title
+					LabelField = "Title",
+					//Array of Categories (see above), mapped to one of the coordinates of the 2D-data array
+					ItemsSource = new Category[]
```
Comment (objorke) [inline models/series/HeatMapSeries.rst]: The Category class is not included in the code.
Can a string array be used instead?

File: models/series/HeatMapSeries.rst [hunk 2]
```diff
@@ -28,9 +25,159 @@ To show the x and y values with one digit, use the format string ``"{2:0.0},{4:0
...
 
-    var model = new PlotModel { Title = "HeatMapSeries" };
+    using OxyPlot;
+    using OxyPlot.Axes;
+    using OxyPlot.Series;
+    
+	namespace ExampleLibrary
+	{
+        
+		public class LinearHeatMapViewModel
```
Comment (objorke) [inline models/series/HeatMapSeries.rst]: this should be the categorized view model?

**Comment by seijikun** [comment]
Argh, I shouldn't create PRs while beeing in a hurry.
Is it possible to pass the categories as `string[]`? I couldn't get it to work and therefore created the `Category` class.
You're right. The class should be called `CategorizedHeatMapViewModel`.
If you could tell me how to do it with a simple `string[]`, I'll fix both issues.

**Comment by objorke** [comment]
I fixed the string[] issue in the CategoryAxis.
Modified your example to follow the coding style of the project. Added it to the documentation-examples project. Updated the pngs from the files generated by the example generator program. 
Fixed the docs build trigger.
The new version can be seen on
http://docs.oxyplot.org/en/latest/models/series/HeatMapSeries.html
Thanks for contributing!

**Pull request merged**